semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/operatorextension/AssignmentTrav.scala

```diff
@@ -1,16 +1,13 @@
 package io.shiftleft.semanticcpg.language.operatorextension
 
 import gremlin.scala.GremlinScala
-
-import io.shiftleft.codepropertygraph.generated.nodes.{Call, Expression}
+import io.shiftleft.codepropertygraph.generated.nodes
 import io.shiftleft.semanticcpg.language.Steps
 
 /**
   * A wrapper for assignment calls that offers syntactic sugar
   * */
-class AssignmentTrav(override val raw: GremlinScala[Call]) extends Steps[Call](raw) {
-
+class AssignmentTrav(override val raw: GremlinScala[nodes.Call]) extends Steps[nodes.Call](raw) {
```

Comment:
Since this is still the old pattern, i think we should extend from `NodeSteps` instead of `Steps` otherwise we wont have steps like `file`.

Reply:
Yup agreed, thanks for spotting. I did that in some other places, however apparently not all. At the end I'll look for all `extends NodeSteps` and `extends Steps` and double check which ones are required.